<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="soccf-tr5-001" name="component-tests(failed)(failed)(failed)(failed)" tests="1" failures="1" timestamp="14 Sep 2014 18:26:50 GMT" time="2.778" errors="0">
  <testcase name="testAll" time="0.124" classname="teammates.test.cases.logic.FeedbackSessionsLogicTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;1&amp;gt; but was:&amp;lt;0&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<1> but was:<0>
	at teammates.test.cases.logic.FeedbackSessionsLogicTest.testGetFeedbackSessionsClosingWithinTimeLimit(FeedbackSessionsLogicTest.java:184)
	at teammates.test.cases.logic.FeedbackSessionsLogicTest.testAll(FeedbackSessionsLogicTest.java:97)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testAll -->
</testsuite> <!-- component-tests(failed)(failed)(failed)(failed) -->
